precompiles/dapps-staking/src/mock.rs

```diff
@@ -81,13 +81,14 @@ impl Default for TestAccount {
     }
 }
 
-impl AddressMapping<TestAccount> for TestAccount {
-    fn into_account_id(h160_account: H160) -> TestAccount {
+// needed for associated type in pallet_evm
+impl AddressMapping<AccountId> for TestAccount {
+    fn into_account_id(h160_account: H160) -> AccountId {
         match h160_account {
-            a if a == H160::repeat_byte(0x11) => Self::Alex,
-            a if a == H160::repeat_byte(0x22) => Self::Bobo,
-            a if a == H160::repeat_byte(0x33) => Self::Dino,
-            _ => Self::Empty,
+            a if a == H160::repeat_byte(0x11) => TestAccount::Alex as u64,
```

Comment:
I'd suggest you keep it consistent with the `AccountId` type. Just call TestAccount::Alex.into();

Reply:
thanks, changed!